### ISSUE TITLE
get_library_playlists Returns Google Play Music Playlists

### ISSUE THREAD

**Issue by oraqol** [sigma67/ytmusicapi#256]
Very strange behavior. I grab the cookie from music.youtube.com, add it to headers_auth.json, but the response does not contain any of the playlists on music.youtube.com, but rather, my old playlists from Google Play Music, to which I no longer have access. I even checked the thumbnail urls for the playlists, and I distinctly remember the icons from Google Play Music. Even weirder, apparently when I run get_liked_songs, that list _does_ contain the liked songs on my current music.youtube.com library. Is anybody else experiencing this issue?

**Comment by sigma67** [comment]
Please provide more details and make sure you're using the right user (top right icon)

**Comment by oraqol** [comment]
Verified that the user on the top right icon was set correctly. I noticed that I have another account in the 'Switch Accounts' section, not a separate google account, but a separate profile on the same google account. When I select that secondary account, I see that the playlists returned by get_library_playlists is returning the playlists from this secondary account, this account also being the one that play music imported my old libraries. I've tried copying the cookie from both profiles into headers_auth.json, both cookies return the same playlists from my old profile.
